packages/sdk-codegen/src/kotlin.gen.ts

```diff
@@ -184,6 +197,49 @@ import java.util.*
     )
   }
 
+  // overridden from CodeGen
+  assignParams(method: IMethod, inputs: ArgValues): string {
+    const args: string[] = []
+    let hasComplexArg = false
+    if (Object.keys(inputs).length > 0) {
+      let requestType: IType | undefined
+      if (
+        !this.useNamedArguments &&
+        (requestType = this.api.getRequestType(method))
+      ) {
+        args.push(this.assignType(this.indentStr, requestType, inputs))
+        hasComplexArg = true
+      } else {
+        method.allParams.forEach((p) => {
+          const v = this.argValue(this.indentStr, p, inputs)
+          if (v !== '') {
+            // const arg = this.useNamedArguments ? `${p.name}=${v}` : v
+            args.push(v)
+            if (!p.type.intrinsic) {
+              hasComplexArg = true
+            }
+          }
+        })
+      }
+    }
+    let open = ''
+    if (args.length > 1 || hasComplexArg) {
+      open = `\n${this.indentStr}`
+    }
+    return args.length > 0 ? `${open}${args.join(this.argDelimiter)}` : ''
+  }
+
+  // overridden from CodeGen
+  makeTheCall(method: IMethod, inputs: ArgValues): string {
+    inputs = trimInputs(inputs)
+    const typeName = method.returnType?.type
+      ? this.typeMap(method.returnType.type).name
+      : 'void'
+    const resp = `val response = await sdk.ok<${typeName}>(sdk.${method.name}(`
+    const args = this.assignParams(method, inputs)
+    return `${resp}${args}))`
```

Comment:
no `prettify` for kotlin either?

Reply:
I'm working on `makeTheCall()` for Kotlin right now but sadly all the "community" prettier plugins I've looked at so far have browser-hostile dependencies. So we're gonna have to accept my not so pretty, but compilable, Kotlin code.

Reply:
There's an officially supported Ruby prettier -- wondering how hard it would be to fork for Python if it's browser-friendly